ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/DebugTraceCall.java

```diff
@@ -69,6 +70,44 @@ protected TraceOptions getTraceOptions(final JsonRpcRequestContext requestContex
     }
   }
 
+  protected Object resultByBlockHeader(
```

Comment:
This method doesn't appear to be used anywhere in DebugTraceCall

Reply:
Hi, yeah, I just removed it, I also removed the `errorResponse` method. Weirdly, the compiler caught that `errorResponse` was not being used anywhere when I tried compiling and building after removing `resultByBlockHeader` right now but didn't catch `resultByBlockHeader`...